pyproject.toml

```diff
@@ -22,6 +22,7 @@ dependencies = [
   "Pillow==10.3.0",
   "jupyterlab==4.0.7",
   "ipywidgets==8.1.1",
+  "pytest==7.2.2",
```

Comment:
Why do we need pytest inside of `pyproject.toml`? We have another file that handles dev dependencies in `tt_metal/python_env/requirements-dev.txt`

Reply:
I missed to delete this code. When I tried the main branch on March 26 there needs the pytest, but after I rebased to the current main branch and test, it don't needs this code.
I have deleted it, thanks for you reviewing.